pkgs/tools/misc/libcpuid/default.nix

```diff
@@ -1,27 +1,16 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, autoreconfHook }:
+{ lib, stdenv, fetchFromGitHub, autoreconfHook }:
 
 stdenv.mkDerivation rec {
   pname = "libcpuid";
-  version = "0.5.1";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
     owner = "anrieff";
     repo = "libcpuid";
     rev = "v${version}";
-    sha256 = "sha256-m10LdtwBk1Lx31AJ4HixEYaCkT7EHpF9+tOV1rSA6VU=";
+    sha256 = "sha256-XX+XFd1ehahIUGONAx2iOxetU0s6vONc7m91zEOWz5o=";
   };
 
-  patches = [
-    # Work around https://github.com/anrieff/libcpuid/pull/102.
-    ./stdint.patch
-    # Fixes cross-compilation to NetBSD, remove me for libcpuid > 0.5.1
-    (fetchpatch {
-      name = "use-popcount-from-libc.patch";
-      url = "https://github.com/anrieff/libcpuid/commit/1acaf9980b55ae180cc08db218b9face28202519.patch";
-      sha256 = "0lvsv9baq0sria1f1ncn1b2783js29lfs5fv8milp54pg1wd5b7q";
-    })
-  ];
-
   nativeBuildInputs = [ autoreconfHook ];
```

Comment:
Is this still needed, if we're not patching anymore?

Reply:
Yes, it still handles the makefile generation / is required.